examples/azure-vm-simple-linux-managed-disk/terraform.tfvars

```diff
@@ -0,0 +1,8 @@
+# Replace with relevant values
+
+# resource_group = "myresourcegroup"
+# rg_prefix      = "rg"
+# hostname       = "myvm"
+# dns_name       = "mydnsname"
+# location       = "southcentralus"
+# admin_password = "T3rr@f0rmP@ssword"
```

Comment:
Sure @tombuildsstuff, we'll remove this file altogether, set defaults in `variables.tf`, and leave sensitive variables (`admin_password`) without a default. 👍